packages/nvidia_nat_langchain/src/nat/plugins/langchain/eval/utils.py

```
@@ -0,0 +1,326 @@
+    module_path, attr_name = parts
+
+    try:
+        module = importlib.import_module(module_path)
+    except ImportError as e:
+        raise ImportError(f"Could not import module '{module_path}' for {label} '{dotted_path}'. "
+                          f"Make sure the package is installed and the path is correct.") from e
+
+    obj = getattr(module, attr_name, None)
+    if obj is None:
+        raise AttributeError(f"Module '{module_path}' has no attribute '{attr_name}'. "
+                             f"Available attributes: {[a for a in dir(module) if not a.startswith('_')]}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)